.cargo/config

```diff
@@ -22,3 +22,6 @@ rustflags = "-Lnative=/usr/local/i686-linux-musl/lib"
 
 [target.aarch64-unknown-linux-musl]
 rustflags = "-Lnative=/usr/local/aarch64-linux-musl/lib"
+
+[target.armv7-unknown-linux-musleabihf]
+rustflags = "-Lnative=/usr/local/arm-linux-musleabihf/lib"
```

Comment:
What motivated this?

Reply:
Essentially, `leveldb-sys` wants to statically link `libstdc++` on `musl`-esque platforms. Since we're not feature flagging the LevelDB-based disk buffer code anymore, compiling on that target triple failed because the static object for `libstdc++` is only located in the aforementioned path, which isn't a standard library search path.

Reply:
Figured. Make sense to me.